dsref/load.go

```diff
@@ -10,14 +10,15 @@ import (
 // ErrNoHistory indicates a resolved reference has no HEAD path
 var ErrNoHistory = fmt.Errorf("no history")
 
-// Loader loads and opens a dataset. The only useful implementation of the
-// loader interface is in github.com/qri-io/qri/lib.
-// TODO(b5) - This interface is a work-in-progress
+// ErrRefNotResolved is the error when LoadResolved is called on a non-resolved ref
+var ErrRefNotResolved = fmt.Errorf("ref has not been resolved, cannot load")
+
+// Loader loads datasets
```

Comment:
love straightforward comments like this. `Loader loads datasets` isn't going to go stale anytime soon 😄